backend/pkg/serde/service.go

```diff
@@ -38,6 +39,7 @@ func NewService(schemaService *schema.Service, protoSvc *proto.Service, msgPackS
 			AvroSerde{SchemaSvc: schemaService},
 			ProtobufSerde{ProtoSvc: protoSvc},
 			ProtobufSchemaSerde{ProtoSvc: protoSvc},
+			RestSerde{restSvc},
```

Comment:
I've added it before `MsgPackSerde` so that message pack can be the fallback if the REST call fails for any reason. But I understand this might be specific to our use case, so let me know if you prefer it in a different order.